src/sequence_batch_scheduler.cc

```diff
@@ -59,7 +59,8 @@ SequenceBatchScheduler::Create(
     const std::unordered_map<std::string, bool>& enforce_equal_shape_tensors,
     std::unique_ptr<Scheduler>* scheduler)
 {
-  std::unique_ptr<SequenceBatchScheduler> sched(new SequenceBatchScheduler());
+  std::unique_ptr<SequenceBatchScheduler> sched(
+      new SequenceBatchScheduler(model, enforce_equal_shape_tensors));
```

Comment:
This appears concerning... Do you know why SequenceBatchScheduler constructor was not forwarded "model" and "enforce_equal_shape_tensors" parameters before?

Reply:
Are they going to be updated across different sequence batch scheduler creation in the past? I think they are not going to be changed across model instance update, unless the model is destructed. Could you elaborate more on why it is concerning?

Reply:
Synced offline: there is no conflict for the instance update case.